sacrebleu/dataset/wmt_xml.py

```diff
@@ -97,6 +97,11 @@ def get_sents(doc):
                 orig_langs.append(origlang)
                 src_sent_count += 1
 
+        # For backward compatibility, if "ref" is not in the fields,
+        # add reference seneteces from the first translator as "ref" field
+        if "ref" not in refs:
+            refs["ref"] = refs[min(refs.keys())]
+
         return {"src": src, **refs, "docid": docids, "origlang": orig_langs,}
```

Comment:
Is this going to work? Don't `fieldnames()` and `get_files()` have to be the same length?

There are also other datasets with multiple refs, e.g., `wmt17/ms`.

Maybe we have to give up this backward compatibility. It's just the CLI, so it's less of an issue.

Reply:
Sure, fieldnames() and get_files() will be the same length. Because method `get_files` will first take all field from method `fieldnames` and then find corresponding files.